provider/validators/cdo_role_test.go

```diff
@@ -1,72 +1,56 @@
-// Copyright (c) HashiCorp, Inc.
-// SPDX-License-Identifier: MPL-2.0
-
 package validators_test
 
 import (
 	"context"
 	"testing"
 
+	"github.com/CiscoDevnet/terraform-provider-cdo/validators"
 	"github.com/hashicorp/terraform-plugin-framework/schema/validator"
 	"github.com/hashicorp/terraform-plugin-framework/types"
-
-	"github.com/hashicorp/terraform-plugin-framework-validators/stringvalidator"
 )
 
-func TestOneOfValidator(t *testing.T) {
+func TestOneOfRolesValidator(t *testing.T) {
 	t.Parallel()
 
+	SuperAdminRoleToken := "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlcyI6WyJST0xFX1NVUEVSX0FETUlOIl19.FBEPzCpXPiYX6esud26WMrJvU3reLDVFdLociGrilVw"
```

Comment:
Just checking: these tokens are now expired, right? If not (i.e., if they are API tokens), please revoke them and use temporary access tokens that we know will expire.

Reply:
Yes, these are fake JWT tokens that only contain the data I need for the tests